Client/Frontend/Widgets/PhotonActionSheet.swift

```diff
@@ -518,11 +508,17 @@ private class PhotonActionSheetCell: UITableViewCell {
     }()
     
     lazy var statusIcon: UIImageView = {
-        let siteImageView = UIImageView()
-        siteImageView.contentMode = .scaleAspectFit
-        siteImageView.clipsToBounds = true
-        siteImageView.layer.cornerRadius = PhotonActionSheetCellUX.CornerRadius
-        return siteImageView
+        let statusIcon = UIImageView()
+        statusIcon.contentMode = .scaleAspectFit
+        statusIcon.clipsToBounds = true
+        statusIcon.layer.cornerRadius = PhotonActionSheetCellUX.CornerRadius
+        statusIcon.setContentHuggingPriority(UILayoutPriorityRequired, for: .horizontal)
+        return statusIcon
+    }()
+
+    lazy var disclosureLabel: UILabel = {
```

Comment:
Any difference between doing this and just `lazy var disclosureLabel = UILabel()` since you're not doing anything else during init?

Reply:
I like to keep the same pattern so when we do decide to add something later we follow it.